### PR TITLE
mircearoata.SatisfactoryModManager version 2.5.0

### PR DESCRIPTION
**Pull request by Galvantua**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19601)

### CODE REVIEW

File: manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: mircearoata.SatisfactoryModManager
```
Comment (OfficialEsco) [inline manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.installer.yaml:4]:
```suggestion
PackageIdentifier: SatisfactoryModding.SatisfactoryModManager
```

File: manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: mircearoata.SatisfactoryModManager
```
Comment (OfficialEsco) [inline manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: SatisfactoryModding.SatisfactoryModManager
```

File: manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: mircearoata.SatisfactoryModManager
```
Comment (OfficialEsco) [inline manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.yaml:4]:
```suggestion
PackageIdentifier: SatisfactoryModding.SatisfactoryModManager
```

File: manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: mircearoata.SatisfactoryModManager
+PackageVersion: 2.5.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/satisfactorymodding/SatisfactoryModManager/releases/download/v2.5.0/Satisfactory-Mod-Manager-Setup.exe
+  InstallerSha256: E8EF0CF3FCB5C96C21A3EB6DE92963AFD045D1F0D62DD4D752EA3AD1873D8EB7
```
Comment (OfficialEsco) [inline manifests/m/mircearoata/SatisfactoryModManager/2.5.0/mircearoata.SatisfactoryModManager.installer.yaml:10]:
```suggestion
- Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://github.com/satisfactorymodding/SatisfactoryModManager/releases/download/v2.5.0/Satisfactory-Mod-Manager-Setup.exe
  InstallerSha256: E8EF0CF3FCB5C96C21A3EB6DE92963AFD045D1F0D62DD4D752EA3AD1873D8EB7
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://github.com/satisfactorymodding/SatisfactoryModManager/releases/download/v2.5.0/Satisfactory-Mod-Manager-Setup.exe
  InstallerSha256: E8EF0CF3FCB5C96C21A3EB6DE92963AFD045D1F0D62DD4D752EA3AD1873D8EB7
  InstallerSwitches:
    Custom: /ALLUSERS
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
Created a proper manifest in #19604 since this will still have multiple issues

**Comment by Galvantua** [comment]
My bad, I wrote it a bit wrong, added the wrong name for the author, and the file name is messed up. How should I go about fixing this?

**Comment by OfficialEsco** [comment]
> My bad, I wrote it a bit wrong, added the wrong name for the author, and the file name is messed up. How should I go about fixing this?

Search before you submit, the Author already added the package so you should have have done `wingetcreate update SatisfactoryModding.SatisfactoryModManager --version --url` instead of `wingetcreate new`.

The Author used the wrong Publisher tho so that had to get fixed anyways, the Publisher Metadata is getting fixed in #19603, you will then see `Satisfactory Mod Mananger` in `winget list` and `winget upgrade`.

For now you can close this PR and wait for #19604 since it got a lot more metadata. Next time you update it there should be no issues unless the Author changes something.

**Pull request closed without merging**